tools/DocumentationBuilder/JsdocPlugin/parser.pegjs

```diff
@@ -48,7 +48,7 @@ docblock
             case "ThisInstruction":
                 if ( docblock.thisParameter ) {
                     var error = new Error("Duplicate @this instruction");
-                    error.offset = instruction.start;
+                    error.location = instruction.start;
```

Comment:
Hmm, I think if you change these you have to also change this: https://github.com/jussi-kalliokoski/trine/blob/9a23d43c3c5a7e9af1970ef9438dbaecc02f9fea/tools/DocumentationBuilder/JsdocPlugin/index.js#L45